scipy/stats/_multivariate.py

```diff
@@ -3320,6 +3297,13 @@ class special_ortho_group_gen(multi_rv_generic):
     This generates one random matrix from SO(3). It is orthogonal and
     has a determinant of 1.
 
+    Alternatively, the object may be called (as a function) to fix the `dim`
```

Comment:
`special_ortho_group` parameter `seed` is not documented.

Reply:
Done.